sale_exception_holidays_public/models/sale_order.py

```diff
@@ -0,0 +1,43 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    def _is_commitment_date_a_public_holiday(self):
+        """
+        Returns True if commitment_date is a public holiday
+        :return: bool
+        """
+        self.ensure_one()
+        res = False
```

Comment:
`res` is not useful as not used after. Return `False` if commitment_date instead